### PR TITLE
Added address(base58) decode to hex command

### PR DESCRIPTION
**Pull request by vsavovlime**

### CODE REVIEW

File: bin/aecli-crypto.js [hunk 1]
```diff
@@ -118,6 +118,20 @@ function unpackTx (tx) {
...
+// This helper function decodes address(base58) to hex
+function decodeAddress (options) {
+  const decoded = Crypto.decodeBase58Check(options.split('_')[1]).toString('hex');
+  console.log(`Decoded address (hex): ${decoded}`)
+}
+
+program
+  .command('decode <base58address>')
+  .description('Decodes base58 address to hex')
+  .option('-a, --address base58address]', 'Address to decrypt', '.')
```
Comment (Perseverance) [inline bin/aecli-crypto.js]: You have a ] at the end of the message

Comment (vsavovlime) [reply bin/aecli-crypto.js]: Removed

File: bin/aecli-crypto.js [hunk 2]
```diff
@@ -118,6 +118,20 @@ function unpackTx (tx) {
...
+// This helper function decodes address(base58) to hex
+function decodeAddress (options) {
+  const decoded = Crypto.decodeBase58Check(options.split('_')[1]).toString('hex');
+  console.log(`Decoded address (hex): ${decoded}`)
+}
+
+program
+  .command('decode <base58address>')
+  .description('Decodes base58 address to hex')
+  .option('-a, --address base58address', 'Address to decrypt', '.')
```
Comment (nduchak) [inline bin/aecli-crypto.js]: Why do you need this options?
Is it the same as command param `.command('decode <base58address>')`?

**Comment by vsavovlime** [comment]
Closed due to @ricricucit  comment

**Pull request closed without merging**